lib/waterfoul/ppu.rb

```diff
@@ -206,14 +206,14 @@ def render_sprites
       39.downto(0) do |sprite|
         sprite_offset = sprite * 4
 
-        sprite_y = $mmu.read_byte(0xFE00 + sprite_offset) - 16
+        sprite_y = $mmu.read_memory_byte(0xFE00 + sprite_offset) - 16
```

Comment:
These `read_memory_byte` seem to improve by a few FPS (< 5 out of 30), so it is some kind of trade-off.